index.html

```diff
@@ -1,312 +1,351 @@
+    rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200&display=swap" rel="stylesheet" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&display=swap" rel="stylesheet">
 
-  <body>
-    <header>
-      <nav>
-        <ul class="nav">
-          <li><a href="/">TUFOIN</a></li>
+  <link rel="stylesheet" href="style.css" />
+  <title>Document</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <div class="nav">
+        <a href="#">TUFOIN</a>
+      </div>
+      <div class="right">
+        <ul class="menuitems hide">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+          <li><a href="/"><i class="fas fa-envelope-square"></i></a></li>
+        </ul>
+        <ul class="menuitems2">
+          <li class="menuitems2 "><a href="/"><i class="fas fa-bars hamburger-icon menu "></i></a></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <main>
+    <section class="main-section">
+      <div class="headline">
+        <h1>
+          Hey There.<br />
+          I'am TUFOIN
+        </h1>
+        <h4>I am a Software Developer</h4>
+        <p>
+          I can help you build a product, feature or website. Look<br> through
+          some of my work and experience! If you like what<br> you see and have a
+          project you need coded,<br> do not hesitate to contact me.
+        </p>
+      </div>
+      <div class="online-icons">
+        <ul class="icons">
+          <li>
+            <a href="/"><i class="fab fa-github"></i></a>
+          </li>
           <li>
-            <a href="/"><i class="fas fa-bars hamburger-icon menu"></i></a>
+            <a href="/"><i class="fab fa-linkedin-in"></i></a>
+          </li>
+          <li>
+            <a href="/"><i class="fab fa-angellist"></i></a>
+          </li>
+          <li>
+            <a href="/"><i class="fab fa-twitter"></i></a>
+          </li>
+          <li>
+            <a href="/"><i class="fab fa-medium-m"></i></a>
           </li>
         </ul>
-      </nav>
-    </header>
+      </div>
+    </section>
+    <div class="work-section">
+      <h1 class="work-h1">My Recent Works</h1>
+      <div class="line"></div>
+    </div>
```

Comment:
- [ ] [OPTIONAL] It seems that you have some issues with this title, see screenshots below 👇 
- Your design
![image](https://user-images.githubusercontent.com/55465133/130269851-51f429a3-4c3e-47c6-b9d0-af4eb813f41f.png)

- Figma design
![image](https://user-images.githubusercontent.com/55465133/130269904-664b8df9-6287-4192-affa-f5c671be0489.png)

Can you please fix this issue?